### PR TITLE
chore(Workspaces): use proxy for db_url

### PR DESCRIPTION
**Pull request by cheikhgwane**
Use proxy for WORKSPACE_DATABASE_URL.

## Changes

Please list / describe the changes in the codebase for the reviewer(s).

- Change 1
- Change 2

## How/what to test

A few words about what needs to be tested, along with specific testing instructions if needed.

## Screenshots / screencast

If relevant, please add some screenshots or a short video.

### CODE REVIEW

File: hexa/workspaces/models.py [hunk 1]
```diff
@@ -152,8 +152,7 @@ def db_port(self):
 
     @property
     def db_url(self):
-        host = get_db_server_credentials()["host"]
-        return f"postgresql://{self.db_name}:{self.db_password}@{host}:{self.db_port}/{self.db_name}"
+        return f"{self.slug}.{settings.WORKSPACES_DATABASE_PROXY_HOST}"
```
Comment (pvanliefland) [inline hexa/workspaces/models.py:155]: @cheikhgwane only the IP should be replaced by xyz.explore.openhexa.org.

By the way, `f"{self.slug}.{..."` could be replaced by `self.db_host` ;)

**Review by pvanliefland — CHANGES_REQUESTED** [review]
@cheikhgwane, see my comment about the url generation.

File: hexa/workspaces/models.py (hunk 1 above)
Comment (cheikhgwane) [reply hexa/workspaces/models.py:155]: Ah yes, my bad.

Comment (cheikhgwane) [reply hexa/workspaces/models.py:155]: Something like this ?
```python
f"postgresql://{self.db_name}:{self.db_password}@{settings.WORKSPACES_DATABASE_PROXY_HOST}:{self.db_port}/{self.db_name}"
```

**Review by pvanliefland — APPROVED** [review]
LGTM!

**Pull request merged**